sirepo/sim_data/flash.py

```diff
@@ -100,31 +101,25 @@ def fixup_old_data(cls, data):
                         'surface area flam=0.9',
                     ]
 
-
     @classmethod
-    def flash_exe_path(cls, data, unchecked=False):
-        from pykern import pkio
-        import distutils.spawn
-        n = '{}-{}'.format(
-            cls._FLASH_PREFIX,
-            data.models.simulation.flashType,
+    def flash_compilation_to_sim_file_basenames(cls, data):
+        # POSIT: values match cls._sim_file_basenames
+        return PKDict(
+            flash4=cls.flash_exe_basename(data),
+            setup_units=cls.flash_setup_units_basename(data),
         )
-        p = distutils.spawn.find_executable(n)
-        if p:
-            return pkio.py_path(p)
-        if unchecked:
-            return  None
-        raise AssertionError(f'unable to find executable={n}')
 
     @classmethod
-    def flash_setup_units_path(cls, data):
-        return cls.flash_exe_path(data).join(
-            '..',
-            '..',
-            'share',
-            cls._FLASH_PREFIX,
-            f'{cls._FLASH_SETUP_UNITS_PREFIX}-{data.models.simulation.flashType}',
-        )
+    def flash_exe_basename(cls, data):
```

Comment:
the flash_exe_basename has little to do with sim_data now. Rather, it should be in a flash template module, because the coupling is increasing to no benefit.

Reply:
The code has changed a bit since you wrote this. Do you still think it is true?

Reply:
It's probably fine to leave it. Not thrilled, but then we'll learn about this as we go. Not much to change if we decide to do something differently.